tests/PKSim.Matlab.Tests/MatlabPopulationFactorySpecs.cs

```diff
@@ -32,17 +36,32 @@ public class When_creating_a_non_preterm_population_from_matlab : concern_for_Ma
       protected override void Context()
       {
          base.Context();
-         _settings = new PopulationSettings
+         var originData = new OriginData
          {
             Species = CoreConstants.Species.HUMAN,
             Population = CoreConstants.Population.ICRP,
-            MinAge = 0,
-            MaxAge = 80,
-            MinWeight = 70,
+         };
+
+         originData.AddCalculationMethods("SurfaceAreaPlsInt_VAR1");
+
+         _settings = new PopulationSettings
+         {
+            Individual = new Individual
```

Comment:
All properties of the Indiivudla can be specified as well here (Seed, Molecules etc..) super powerful feature